src/en/glossary.md

```diff
@@ -2,17 +2,18 @@ Title: Glossary
 
 # Glossary of terms
```

Comment:
This glossary page is not exposed any more. this is another legacy issue - it wasn't deleted because a lot of the authors-* pages refer to it and it would therefore break links. 

The *actual* glossary page is https://jujucharms.com/docs/2.1/juju-concepts

I suppose it doesn't hurt to change it, but it will be deleted anyhow someday